examples/react-wagmi/package.json

```diff
@@ -8,7 +8,6 @@
   },
   "dependencies": {
     "@reown/appkit": "workspace:*",
-    "@reown/appkit-adapter-wagmi": "workspace:*",
```

Comment:
we should do a different example, will modify this

Reply:
What do you mean? Doesn't the basic single adapter examples work now?

Reply:
yes, what I mean we should not modify this one in particular